.gitignore

```diff
@@ -16,3 +16,4 @@
 
 # pathfinder variables env file
 pathfinder-var.env
```

Comment:
This line should be removed I think now that you've modified `docker-compose.yml` not to use it.

Reply:
note: I used `.env` instead because `docker compose` automatically reads from it